src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -183,6 +183,17 @@ foam.CLASS({
           });
         }
       }
+    },
+    {
+      name: 'createUserDefaultLanguage',
+      code: function() {
+        let l = foam.locale.split("-")
+        let code = l[0]
+        let varient = l[1] ? l[1] : undefined
```

Comment:
Could just be varient = l[1] || undefined;, or because you just use it for its truth-iness, you could just leave it as variant = l[1]

Reply:
You could also, just grab foam.language, except that it seems to be set incorrectly.